src/main/java/com/twilio/oai/TwilioCsharpGenerator.java

```diff
@@ -243,7 +243,11 @@ private void requestBodyArgument(final CodegenOperation co) {
         co.requiredParams.forEach(parameter -> requestBodyArgument.put(parameter.paramName, parameter));
         co.pathParams.forEach(parameter -> requestBodyArgument.put(parameter.paramName, parameter));
         conditionalParameters.forEach(parameter -> requestBodyArgument.put(parameter.paramName, parameter));
-        optionalParameters.forEach(parameter -> requestBodyArgument.put(parameter.paramName, parameter));
+        if (co.operationId.startsWith("List") || co.operationId.startsWith("Fetch")) {
+            optionalParameters.forEach(parameter -> requestBodyArgument.put(parameter.paramName, parameter));
+        }
+        co.formParams.forEach(parameter -> requestBodyArgument.put(parameter.paramName, parameter));
```

Comment:
This can be improved later. If optionalParameters is a superset of formParams and headerParams, the second section can be covered in an else loop